### PR TITLE
InvariantCulture to fix localization

### PR DESCRIPTION
**Pull request by erwinkramer**
# Description

Use `InvariantCulture` to fix localization.

Or else you get (on non-US): 

```
== APP == insert into orders (orderid, customer, price) values (1, 'John Smith', 100,32);
== APP == fail: Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware[1]
== APP ==       An unhandled exception has occurred while executing the request.
== APP ==       Dapr.DaprException: Binding operation failed: the Dapr endpoint indicated a failure. See InnerException for details.
== APP ==        ---> Grpc.Core.RpcException: Status(StatusCode="Internal", Detail="error invoking output binding sqldb: error executing query: ERROR: INSERT has more expressions than target columns (SQLSTATE 42601)")`
```

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] The quickstart code compiles correctly
* [x] You've tested new builds of the quickstart if you changed quickstart code
* [x] You've updated the quickstart's README if necessary
* [x] If you have changed the steps for a quickstart be sure that you have updated the automated validation accordingly. All of our quickstarts have annotations that allow them to be executed automatically as code. For more information see [mechanical-markdown](https://github.com/dapr/mechanical-markdown#readme). For user guide with examples see [Examples](https://github.com/dapr/mechanical-markdown/tree/main/examples#mechanical-markdown-by-example).

### CODE REVIEW

File: bindings/csharp/http/batch/program.cs [hunk 1]
```diff
@@ -30,18 +30,23 @@
...
 httpClient.DefaultRequestHeaders.Accept.Add(new System.Net.Http.Headers.MediaTypeWithQualityHeaderValue("application/json"));
 
 // Triggered by Dapr input binding
-app.MapPost("/" + cronBindingName, async () => {
+app.MapPost("/" + cronBindingName, async () =>
+{
      Console.WriteLine("Processing batch..");
 
+     // Ensure the culture is set to one that uses dot notation
+     CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;
```
Comment (yaron2) [inline bindings/csharp/http/batch/program.cs:44]: Isn't this better set one time outside the function handler?

Comment (erwinkramer) [reply bindings/csharp/http/batch/program.cs]: Yes you're right, but that requires implementing it inside the `WebApplicationBuilder`, which i did with the latest commit!

**Review by paulyuk — APPROVED** [review]
LGTM.  Thank you for contribution and adjustments to the builder.  Running tests now against release-1.14 so we can take it there.

**Comment by paulyuk** [comment]
closed and reopened to retest on rebase of `release-1.14` branch

**Comment by paulyuk** [comment]
This only affects C# quickstarts, and those checks pass.  I'm merging.

**Pull request merged**